packages/react/src/themes/teams/components/Icon/svg/icons/filesCode.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g>
+        <path d="M20.5 8H15c-.4 0-.777.156-1.083.463l-3.478 3.968c-.283.283-.439.66-.439 1.06V22.5c0 .827.673 1.5 1.5 1.5h9c.827 0 1.5-.673 1.5-1.5v-13c0-.827-.673-1.5-1.5-1.5zm-6.514 1.9V13h-2.718l2.718-3.1zM21 22.5a.5.5 0 0 1-.5.5h-9a.5.5 0 0 1-.5-.5V14h3.986V9.003c.005 0 .01-.003.014-.003h5.5a.5.5 0 0 1 .5.5v13zm-2.462-6.24a.375.375 0 0 0-.576.48l1.05 1.26-1.05 1.26a.375.375 0 1 0 .576.48l1.25-1.5a.375.375 0 0 0 0-.48l-1.25-1.5zm-4.548-.048a.375.375 0 0 0-.528.048l-1.25 1.5a.375.375 0 0 0 0 .48l1.25 1.5a.374.374 0 1 0 .576-.48L12.988 18l1.05-1.26a.375.375 0 0 0-.048-.528zm3.129-1.568a.374.374 0 0 0-.475.237l-2 6a.375.375 0 1 0 .712.238l2-6a.375.375 0 0 0-.237-.475z" />
+        <path d="M22.2 6h-8c-.5 0-.9.2-1.3.5l-4.4 5c-.3.4-.5.8-.5 1.3v11.4c0 1 .8 1.8 1.8 1.8h12.5c1 0 1.8-.8 1.8-1.8V7.8c-.1-1-.9-1.8-1.9-1.8zM13 8v4H9.5L13 8zm10 16.2c0 .4-.3.8-.8.8H9.8c-.5 0-.8-.3-.8-.8V13h5V7h8.2c.5 0 .8.3.8.8v16.4zm-3.9-8c-.2-.2-.5-.2-.7-.1-.2.2-.2.5-.1.7l1.3 1.7-1.3 1.7c-.2.2-.1.5.1.7.1.1.2.1.3.1.1 0 .3-.1.4-.2l1.6-2c.1-.2.1-.5 0-.6l-1.6-2zm-5.6-.1c-.2-.2-.5-.1-.7.1l-1.6 2c-.1.2-.1.5 0 .6l1.6 2c.1.1.2.2.4.2.1 0 .2 0 .3-.1.2-.2.2-.5.1-.7l-1.3-1.7 1.3-1.7c.2-.2.1-.5-.1-.7zm3.9-2.1c-.3-.1-.5.1-.6.3l-2.5 8c-.1.3.1.5.3.6h.2c.2 0 .4-.1.5-.3l2.5-8c0-.2-.2-.5-.4-.6z" />
```

Comment:
remove this path. it's a remnant from the old way we had icons and isn't needed for this.

Reply:
fixed! thanks!